.buildkite/release-automation/pre_release.rayci.yml

```diff
@@ -0,0 +1,36 @@
+group: Pre-release checks
+steps:
+  - label: "Check release blockers"
+    key: check-release-blockers
+    depends_on: forge
+    job_env: forge
+    instance_type: small_branch
+    commands:
+      - bazel run //ci/ray_ci/automation:weekly_green_metric -- --check
+
+  - block: "Trigger tests & image builds"
+    depends_on:
+      - step: check-release-blockers
+        allow_failure: true
+
+  - label: "Trigger Postmerge test"
+    trigger: "postmerge"
+    if: build.env("COMMIT") != null && build.env("BRANCH") != null
+    build:
+      commit: "${COMMIT}"
```

Comment:
Instead of having users input their own `COMMIT` env var in the build, I think we can just set the commit & branch when creating build and use `$BUILDKITE_COMMIT` instead here.
That also helps maintain the same commit hash across the whole pipeline build (as `HEAD` is a moving target) which is also used for downloading S3 wheels & as `RAY_HASH` when sanity check the wheels.
For scheduled run of this pipeline, commit is just set to `HEAD` automatically.
What do you think? @aslonnie

Reply:
yes, you should just use `BUILDKITE_COMMIT` and `BUILDKITE_BRANCH`

Reply:
make sure `BUILDKITE_COMMIT` is not set to `HEAD` but actually a specific commit.

Reply:
Doesn't `BUILDKITE_COMMIT` get updated into an actual commit hash after `init` step, like in [this build](https://buildkite.com/ray-project/release-automation/builds/171#018df17d-7101-4ae1-9f19-2e7db5eb3b43/155-565)? All other steps have `BUILDKITE_COMMIT=<commit_hash>` instead of `HEAD` like `init` step.

Reply:
https://buildkite.com/docs/pipelines/environment-variables#BUILDKITE_COMMIT

Reply:
you can add a check step to check that the value of `$BUILDKITE_COMMIT` is the same as `$(giv rev-parse HEAD)` and fail otherwise.

Reply:
done!